routers/api/v1/repo/release_attachment.go

```diff
@@ -19,7 +19,7 @@ import (
 
 // GetReleaseAttachment gets a single attachment of the release
 func GetReleaseAttachment(ctx *context.APIContext) {
-	// swagger:operation GET /repos/{owner}/{repo}/releases/{id}/assets/{attachment_id} repository repoGetReleaseAttachment
+	// swagger:operation GET /repos/{owner}/{repo}/releases/{id}/assets/{asset} repository repoGetReleaseAttachment
```

Comment:
I still think attachment_id is right here.

1. It was documented as so, it's a bug in code so no client should ever work.
2. It is an ID (integer)
3. The code is doing `GetAttachmentByID(ctx, attachID)`

Reply:
I think there is an inconsistent in the previous code. the router in api.go use the parameter variable `asset` but the document using `attachment_id`. If you think change all the places to `attachment_id` is better, I can do that.

Reply:
Personally I prefer "attachment_id" everywhere, it's more consistent, it won't break the legacy code or generated client API code.

If it's changed to "asset", the API itself changes, the generated API client code would also change?